conformance/lang/expressions/field-access-expression/optional_field_access_expr_lax_type.balt

```diff
@@ -0,0 +1,1003 @@
+Test-Case: output
+Description: Test field access expression by accessing existing fields when they belong to JSON objects. Here, the
+             field-names are string-literals.
+Labels: array-type, boolean, decimal, DecimalFloatingPointNumber, DecimalNumber, field-access-expr, float,
+        FloatingPointTypeSuffix, HexFloatingPointLiteral, HexIntLiteral, json, list-constructor-expr,
+        mapping-constructor-expr, nil-literal, readonly-type, string
+
```

Comment:
Don't we need t=to add `optional-field-access-expr` label?

Reply:
My bad. Will update `field-access-expr` to `optional-field-access-expr` for these tests. Thanks.